.buildkite/steps/check-code-committed.sh

```diff
@@ -34,4 +34,9 @@ if ! git diff --no-ext-diff --exit-code; then
   exit 1
 fi
 
+# While we're cleaning up things found by golangci-lint, don't fail if it finds
+# things.
+echo +++ :go: Running golangci-lint...
+golangci-lint run || true
```

Comment:
ooh nice, lets see how noisy it is with the default linters!

Reply:
Okay, thats good - gives us some stuff to work through. Having a scan of the erros, a lot of the staticcheckones are 🤷 and we can probably disable the rules. The errcheck ones are useful though!

```
106 issues:
--
  | * errcheck: 50
  | * govet: 4
  | * ineffassign: 1
  | * staticcheck: 39
  | * unused: 12

```